Book_apis/urls.py

```diff
@@ -23,7 +25,9 @@
     path('createpublisher/', CreatePublisherAPIView.as_view()),
     path('createbook/', CreateBookAPIView.as_view()),
     path('books/', BookDataAPIView.as_view()),
-    #path('createcart/', CreateCartItemAPIView.as_view()),
-    #path('cartitems/', CartDataAPIView.as_view()),
+    path('createstock/', CreateStockAPIView.as_view()),
```

Comment:
While defining the endpoints please do not use plural names also do not use name like "createbook", "createstock".
It should be like "book", "stock" and the POST / Create or GET, PUT method will define the action.

Reply:
path('additem/', CreateCartItemAPIView.as_view()),
    path('cartitems/', CartItemsAPIView.as_view()),
    
    we don't need to create separate classes for POST and GET we can use the same class even with pagination.